main.py

```
@@ -733,6 +778,179 @@ async def create_support_ticket(request: Request, ticket: SupportTicket):
+                        "success": True,
+                        "admin": {
+                            "id": admin_data["admin_id"],
+                            "name": admin_data["admin_name"],
+                            "email": admin_data["email"]
+                        },
+                        "expiresAt": admin_data["expires_at"]
+                    }
+                else:
+                    # Token expired, remove from cache
+                    del authenticated_admins[token]
+            except:
```

Ruff (0.15.4)

[error] Do not use bare `except`

(E722)